src/lib/shared/ui/optimal-image/tailwind-weserv-interop.ts

```diff
@@ -0,0 +1,61 @@
+const width = /\bw-([0-9]+)\b/;
+const height = /\bh-([0-9]+)\b/;
```

Comment:
Maybe append `Pattern` or something to the name? It's a little confusing since these variables do not actually hold the width/height.

Reply:
I thought about it, but do you think it's necessary? I think that the code that uses it is relatively close together

Reply:
Maybe not _necessary_ (which is why it's a "comment" and not "requested change"), but it will aid in readability

Reply:
it's not an "approval" either :p fixed